plugin/serverlessAnalyzeBundle.ts

```diff
@@ -48,5 +40,16 @@ export class ServerlessAnalyzeBundlePlugin implements Plugin {
         await this.bundleVisualizer();
       },
     };
+
+    // The plugin requires metafile option to be true
+    // We set it here (and not within a hook) because Serverless Framework keeps copies of service configuration
+    // therefore overwriting changes made by hook functions
+    //
```

Comment:
We can remove this useless comment line

Reply:
For what is worth, the empty line was intentional (sort of a separator to clearly mark it as a block comment).
Anyway, your repo, I removed it and forcepushed